src/main/java/org/folio/rest/impl/ItemStorageAPI.java

```diff
@@ -39,10 +40,10 @@ public void getItemStorageItems(
   @Validate
   @Override
   public void postItemStorageItems(
-      String lang, Item entity,
-      RoutingContext routingContext, Map<String, String> okapiHeaders,
-      Handler<AsyncResult<Response>> asyncResultHandler,
-      Context vertxContext) {
+    String lang, Item entity,
+    RoutingContext routingContext, Map<String, String> okapiHeaders,
+    Handler<AsyncResult<Response>> asyncResultHandler,
+    Context vertxContext) {
```

Comment:
If possible, in future, please could reformatting be done in a separate pull request. It helps make it easier to distinguish the changes relating to the feature from other changes.